### DISCUSSION TITLE
iOS release timeline?

### DISCUSSION THREAD

**Discussion by kashav** [lichess-org/mobile#788] [General]
Is there a release schedule or timeline that I can follow for an iOS release? Is it possible to start submitting beta releases of the new app to the iOS app store?

I played around with the new app in the iPhone simulator and enjoyed it. Would love to get this on my own phone without building it myself, and the previous app hasn't been updated since Dec 2022.

**Comment by kashav** [comment]
Would love to contribute if there is a readily available list of issues that are blocking an iOS release.

**Comment by HaonRekcef** [comment]
I have no idea what is causing the delay in the iOS release, but for now, you can download it via TestFlight: https://testflight.apple.com/join/WEurB7MT

**Comment by kashav** [comment]
Awesome, thanks, was able to download it from TestFlight.
